pkg/util/github/github.go

```diff
@@ -159,17 +159,17 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	out, err := os.Create(assetTmpFile)
 	defer out.Close()
 	if err != nil {
-		return errors.Wrapf(err, "Unable to create file %s", assetTmpFile)
+		return errors.Wrapf(err, "Cannot create file %s", assetTmpFile)
 	}
 
 	// Copy the asset and verify its hash
 	_, err = io.Copy(out, asset)
 	if err != nil {
-		return errors.Wrapf(err, "Failed to copy %s to %s", assetTmpFile, tmpDir)
+		return errors.Wrapf(err, "Cannot copy %s to %s", assetTmpFile, tmpDir)
```

Comment:
> On a grammatical level the two phrases are equivalent,

How can two sentences be equivalent on a grammatical level? They might be grammatically correct in both cases, but imo they differ semantically. 

> "Unexpected error occured while coping x to y",

Sure, something like this works